game_manager:
  max_has_work: 5
